### PR TITLE
Added builder package and composer installer

### PR DESCRIPTION
**Pull request by bkuhl**
- Added builder package
 - Added base fpm `Dockerfile` for use in future CI setup

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,8 @@
     "require-dev": {
         "fzaninotto/faker": "~1.4",
         "phpunit/phpunit": "~4.0",
-        "vluzrmos/tinker": "^1.1"
+        "vluzrmos/tinker": "^1.1",
+        "realpage/builder": "dev-master"
```
Comment (jlaswell) [inline composer.json]: Sounds like we need to tag the repo. I really don't like the idea of running our stuff as `dev-master`.

Comment (bkuhl) [inline composer.json]: t's currently in a highly volatile state is the only reason I left that there.  There's more feedback I'd like to add to the builder package before considering this complete

File: infrastructure/fpm/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM realpage/php:7.0-fpm
+
+ADD . /var/www/html
+
+VOLUME ["/var/www/html"]
```
Comment (jlaswell) [inline infrastructure/fpm/Dockerfile]: The volume is already a part of the base container, so we don't need to recreate a mount. Plus, docker-compose handles exposing that volume for development.

File: composer.json (hunk 1 above)
Comment (bkuhl) [inline composer.json]: If you'd prefer, I could begin versioning the builder

Comment (jlaswell) [inline composer.json]: Yea, I think even a `~0.1` release is good here.

File: infrastructure/fpm/Dockerfile (hunk 2 above)
Comment (bkuhl) [inline infrastructure/fpm/Dockerfile]: Are you suggesting that this is in no way needed?  I was under the impression once this container is deployed to Rancher, this volume entry would be beneficial.  Re-reading the `Dockerfile` documentation I can't find what gave me that impression...

Comment (bkuhl) [inline infrastructure/fpm/Dockerfile]: This has been removed

File: composer.json (hunk 1 above)
Comment (bkuhl) [inline composer.json]: This has been updated to `~0.1`

**Comment by jlaswell** [comment]
LGTM

**Pull request merged**